core/domain/learner_playlist_services_test.py

```diff
@@ -100,62 +101,76 @@ def _get_all_learner_playlist_collection_ids(self, user_id):
             learner_playlist_model.collection_ids if
             learner_playlist_model else [])
 
-    def test_mark_exploration_to_be_played_later(self):
+    def test_already_subscribed_to_exploration(self):
         self.assertEqual(
             self._get_all_learner_playlist_exp_ids(self.user_id), [])
 
-        # Add an exploration to the learner playlist.
+        # Subscribe to exploration.
+        subscription_services.subscribe_to_exploration(
+            self.user_id, 'SUBSCRIBED_ID')
+
+        # Now if we try to add the same exploration, it shouldn't be added.
         learner_progress_services.add_exp_to_learner_playlist(
-            self.user_id, self.EXP_ID_0)
+            self.user_id, 'SUBSCRIBED_ID')
+        self.assertEqual(
+            self._get_all_learner_playlist_exp_ids(
+                self.user_id), [])
+
+
+    def test_mark_exploration_to_be_played_later_with_position(self):
```

Comment:
Again, make the test name clearer and state the expected outcome. See comment above.

Reply:
Got it.